bucket/gimp.json

```diff
@@ -8,48 +8,52 @@
     "innosetup": true,
     "installer": {
         "script": [
+            "$scriptver = ($fname -split '-' | Select-Object -First 1 -Skip 1).Split('.')",
+            "$shortver = $scriptver[0] + '.' + $scriptver[1]",
             "Push-Location \"$dir\"",
             "Get-ChildItem -Filter '*.debug' -Recurse | Remove-Item -Recurse",
             "if ($architecture -eq '64bit') {",
-            "   Rename-Item 'lib\\gimp\\2.0\\plug-ins\\twain\\twain,2.exe' 'twain.exe'",
-            "   Remove-Item 'lib\\gimp\\2.0\\plug-ins\\twain\\twain,1.exe'",
             "   Get-ChildItem -Filter '*,1*' -Recurse | Rename-Item -NewName { $_.name -Replace ',\\d','' }",
             "   Get-ChildItem -Filter '*,*' -Recurse | Remove-Item",
             "} else {",
-            "   Rename-Item 'lib\\gimp\\2.0\\plug-ins\\twain\\twain,1.exe' 'twain.exe'",
-            "   Remove-Item 'lib\\gimp\\2.0\\plug-ins\\twain\\twain,2.exe'",
             "   Get-ChildItem -Filter '*,1*' -Recurse | Remove-Item",
             "   Get-ChildItem -Filter '*,*' -Recurse | Rename-Item -NewName { $_.name -Replace ',\\d','' }",
             "}",
-            "$pyenv = Get-Content 'lib\\gimp\\2.0\\environ\\pygimp.env' -Raw",
-            "$pyenv + '__COMPAT_LAYER=HIGHDPIAWARE' | Set-Content 'lib\\gimp\\2.0\\environ\\pygimp.env'",
-            "$pyint = Get-Content 'lib\\gimp\\2.0\\interpreters\\pygimp.interp' -Raw",
+            "$defpath = \"`nPATH=`${gimp_installation_dir}\\bin`n\"",
+            "$defenv = Get-Content \"lib\\gimp\\$shortver\\environ\\default.env\" -Raw",
+            "$defenv += $defpath",
+            "$defenv += \"PYTHONPATH=`${gimp_installation_dir}\\lib\\gimp\\$shortver\\python;`${gimp_plug_in_dir}\\plug-ins\\python-console\"",
+            "$defenv | Set-Content \"lib\\gimp\\$shortver\\environ\\default.env\"",
+            "# Only worked for versions <2.99, installer no longer supplies 'pygimp.env'",
+            "# $pyenv = (Get-Content \"lib\\gimp\\$shortver\\environ\\pygimp.env\" -Raw) + '__COMPAT_LAYER=HIGHDPIAWARE'",
+            "# $pyenv | Set-Content \"lib\\gimp\\$shortver\\environ\\pygimp.env\"",
+            "'__COMPAT_LAYER=HIGHDPIAWARE' | Set-Content \"lib\\gimp\\$shortver\\environ\\pygimp.env\"",
+            "$pyint = Get-Content \"lib\\gimp\\$shortver\\interpreters\\pygimp.interp\" -Raw",
             "$pyint = $pyint -Replace '(python|python2)=(.*)', \"`$1=$dir\\bin\\pythonw.exe\"",
             "$pyint = $pyint -Replace 'py::python2', 'py::python'",
-            "$pyint | Set-Content 'lib\\gimp\\2.0\\interpreters\\pygimp.interp'",
-            "Pop-Location",
-            "Copy-Item \"$bucketsdir\\extras\\scripts\\gimp\\default.env\" \"$dir\\lib\\gimp\\2.0\\environ\\\" | Out-Null"
+            "$pyint | Set-Content \"lib\\gimp\\$shortver\\interpreters\\pygimp.interp\"",
+            "Pop-Location"
```

Comment:
I experimented with leaving this file as default, and everything seems to work fine. There is no reference to python2 anymore (Gimp 3.0 is using python 3.12), and adding absolute paths seems unnecessary. 

For reference, here is the default file contents extracted from the installer. 

```python=pythonw.exe
python3=pythonw.exe
/usr/bin/python=pythonw.exe
/usr/bin/python3=pythonw.exe
:Python:E::py::python3:
```